security/advancedtls/advancedtls.go

```diff
@@ -251,85 +281,75 @@ func (c *advancedTLSCreds) OverrideServerName(serverNameOverride string) error {
 }
 
 // The function buildVerifyFunc is used when users want root cert reloading, and possibly custom
-// server authorization check.
+// verification check.
 // We have to build our own verification function here because current tls module:
 // 1. does not have a good support on root cert reloading
```

Comment:
Please add a "period" at the end of the of comment sentences. The same applies to other comments in the code. Thanks!

Reply:
Done.